packages/replay/src/replay.ts

```diff
@@ -839,4 +823,33 @@ export class ReplayContainer implements ReplayContainerInterface {
       saveSession(this.session);
     }
   }
+
+  /** Handler for rrweb.record.onMutation */
+  private _onMutationHandler = (mutations: unknown[]): boolean => {
+    const count = mutations.length;
+
+    const fullSnapshotOnMutationsOver = this._options._experiments.fullSnapshotOnMutationsOver || 0;
+
+    // Create a breadcrumb if a lot of mutations happen at the same time
+    // We can show this in the UI as an information with potential performance improvements
+    if (count > 500 || (fullSnapshotOnMutationsOver && count > fullSnapshotOnMutationsOver)) {
```

Comment:
Should we bump this to 2500? @billyvg

Reply:
Hmm wdyt about making it an experiment option (or replay option) with a default of 1000? I'm thinking lower is better so we can alert early, but allow user to override it if they deem the performance to be fine.

Reply:
sounds reasonable!